src/app/utils/trimLongString.ts

```diff
@@ -9,3 +9,5 @@ export function trimLongString(value: string, trimStart = 10, trimEnd = 8, ellip
 
   return `${value.slice(0, trimStart)}${ellipsis}${trimEnd ? value.slice(-trimEnd) : ''}`
 }
+
+export const defaultTrim = (value: string) => trimLongString(value, 6, 6)
```

Comment:
The default trim is `trimStart = 10, trimEnd = 8, ellipsis = '...'`

This is an alternative default? :smile: